guides/14_Content_Security_Policy.md

```diff
@@ -0,0 +1,38 @@
+# Content Security Policy for applications using the Amazon Chime SDK
+
+Modern web applications use [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) to protect users from certain classes of attacks. You can add a security policy by configuring your web server to return the Content-Security-Policy HTTP header or using <meta> element to configure a policy. A guide to use a Lambda@Edge function to inject security headers through CloudFront, see [Adding HTTP Security Headers Using Lambda@Edge and Amazon CloudFront](https://aws.amazon.com/blogs/networking-and-content-delivery/adding-http-security-headers-using-lambdaedge-and-amazon-cloudfront/).
+
+## Meetings connectivity
+
+All applications that use voice and video connectivity — _e.g._, `AudioVideoFacade` classes — will need:
+
+`connect-src`: `*.chime.aws` to reach signaling servers.
+
+## Messaging
+
+* Cognito: 
+
+You can use Amazon Cognito for user authentication when using the Amazon Chime SDK messaging feature. For an example of how to use Amazon Cognito with the Amazon Chime SDK messaging features, see [Build chat features into your application with Amazon Chime SDK messaging](https://aws.amazon.com/blogs/business-productivity/build-chat-features-into-your-application-with-amazon-chime-sdk-messaging/). 
```

Comment:
I don't think you need to explain the Cognito integration in this doc. Just say:

> If you use the [Amazon Chime SDK messaging](https://docs.aws.amazon.com/chime/latest/dg/using-the-messaging-sdk.html) with the [Amazon Cognito](https://aws.amazon.com/cognito/), ensure that you add the following:
>
> `connect-src` `*.amazonaws.com`.

Reply:
Make sense. Will fix.